sunpy/net/dataretriever/sources/tests/test_noaa.py

```diff
@@ -175,6 +184,13 @@ def test_srs_current_year():
     assert res.data[0].endswith(f"{year}0101SRS.txt")
 
 
+@pytest.mark.remote_data
+def test_srs_save_path(tmpdir):
+    qr = Fido.search(a.Instrument.srs_table, a.Time("2016/10/01", "2016/10/02"))
+    files = Fido.fetch(qr, path=str(tmpdir))
+    assert len(files) == 2
+
```

Comment:
Might be worth checking the files have the expected dates too?

Reply:
good point